web/app/view/Report.js

```diff
@@ -67,17 +67,50 @@ Ext.define('Traccar.view.Report', {
         }]
     },
 
-    listeners: {
-        selectionchange: 'onSelectionChange'
+    layout: {
+        type: 'fit'
```

Comment:
I think `card` layout is more appropriate. Also, you don't need curly braces, you can just use string straight away.